lib/python/rose/apps/comparisons/output_grepper.py

```diff
@@ -16,29 +16,30 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 """Return a list of values matching a regular expression."""
 
 from rose.apps.rose_ana import DataLengthError, data_from_regexp
 import re
 
 REGEXPS = {
-  'um_wallclock' : r"Maximum Elapsed Wallclock Time:\s*(\S+)",
-  'um_initial_norms' : r"initial\s*Absolute\s*Norm\s*:\s*(\S+)",
-  'um_final_norms' : r"Final\s*Absolute\s*Norm\s*:\s*(\S+)", }
+    'um_wallclock': r"Maximum Elapsed Wallclock Time:\s*(\S+)",
+    'um_initial_norms': r"initial\s*Absolute\s*Norm\s*:\s*(\S+)",
+    'um_final_norms': r"Final\s*Absolute\s*Norm\s*:\s*(\S+)"}
+
```

Comment:
I am surprised you would both say this, given that closing-brace-on-separate-line was my old preferred style, borrowed from languages that demand open and close braces for blocks. It is certainly a personal taste in Python code, when blocks are done by indentation. I agree that closing-brace-on-same style is probably less readable, but I think it looks prettier in Python code.

PEP8 does not appear to complain either way.